docs/customize/users.md

```diff
@@ -0,0 +1,219 @@
+# Adding Users to Invenio ILS instance
+
+This guide provides step-by-step instructions for adding users to a custom Invenio App ILS instance using:
+
+- An existing database
+- LDAP synchronization
+
+!!! Note
+    The following guides are just an example to give you an idea of how you can get started on existing users to your Invenio ILS instance!
+
+    Similar scripts can also be used to port other cataloging data like Documents, E-Items, Purchase Orders and more!
+
+This process is suitable for administrators who have just set up a new instance and want to onboard all users to the ILS database.
+
+### Prerequisites
+
+- A running instance of Invenio App ILS.
+- Administrative access to the Invenio App ILS instance.
+- Python installed on your system.
```

Comment:
If I have the first, the second prerequisite must be there... I would remove the Python part